src/v1/index.ts

```diff
@@ -35,6 +35,8 @@ import * as testLab from "./providers/testLab";
 
 import { setApp as setEmulatedAdminApp } from "../common/app";
 
+export { traceContext } from "../common/trace";
+
```

Comment:
let's not add any new "feature" in 1st Gen namespace - not technically necessary but more of in the name of principle.